arangod/Agency/State.cpp

```diff
@@ -347,6 +347,7 @@ void State::logEmplaceBackNoLock(log_t&& l) {
   }
 
   try {
+    _log_size += l.entry->byteSize();
```

Comment:
Can this now count a log entry multiple times?
For example, the method directly before it does:
```
  logEmplaceBackNoLock(log_t(idx, term, buf, clientId, millis));
  _log_size += byteSize;
```
meaning that we will call the size of that log entry multiple times.

Reply:
This has not been altered. And I would hope that we could collect all general remarks into a separate PR, that would make its way into all versions.